framework/src/components/storage/adapters/pgp_adapter.js

```diff
@@ -90,7 +90,7 @@ class PgpAdapter extends BaseAdapter {
 
 		this.options.user = this.options.user || process.env.USER;
 
-		this.pgp.end();
+		// this.pgp.end();
```

Comment:
?

Reply:
This was needed when sharing the same instance of Storage across modules as one module was ending the connections of the other. Now we use different instances of Storage  for Chain and API so this should be uncommented back.

Will change it. Thanks for pointing it out.

Reply:
Had to change it back again as parallel tests now fail due to this.

Reply:
All tests run in parallel that may be case of this behaviour. Will look into further while working on multi-process.